### ISSUE TITLE
Invalid array length error with 1.21.2

### ISSUE THREAD

**Issue by ggascoigne** [mac-s-g/react-json-view#340]
v 1.21.2 is giving me an Invalid array length error, reverting to 1.21.1 fixes it.  I can't say that the stack is particularly helpful but perhaps it contains some clue.

```
Uncaught RangeError: Invalid array length
    at n.value (main.js:1)
    at finishClassComponent (react-dom.development.js:17485)
    at updateClassComponent (react-dom.development.js:17435)
    at beginWork (react-dom.development.js:19073)
    at HTMLUnknownElement.callCallback (react-dom.development.js:3945)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:3994)
    at invokeGuardedCallback (react-dom.development.js:4056)
    at beginWork$1 (react-dom.development.js:23964)
    at performUnitOfWork (react-dom.development.js:22776)
    at workLoopSync (react-dom.development.js:22707)
    at renderRootSync (react-dom.development.js:22670)
    at performSyncWorkOnRoot (react-dom.development.js:22293)
    at react-dom.development.js:11327
    at unstable_runWithPriority (scheduler.development.js:646)
    at runWithPriority$1 (react-dom.development.js:11276)
    at flushSyncCallbackQueueImpl (react-dom.development.js:11322)
    at workLoop (scheduler.development.js:590)
    at flushWork (scheduler.development.js:545)
    at MessagePort.performWorkUntilDeadline (scheduler.development.js:157)
The above error occurred in the <n> component:

    at n (http://localhost:3000/static/js/0.chunk.js:5327:16)
    at div
    at div
    at n (http://localhost:3000/static/js/0.chunk.js:5594:9)
    at div
    at n (http://localhost:3000/static/js/0.chunk.js:5766:16)
    at span
    at Suspense
```

**Comment by youngbob** [comment]
I'd guess it's related to #336 - if an object has more than `groupArraysAfterLength` keys, it would be rendered as an array, I think.

**Comment by youngbob** [comment]
Easy step to reproduce: add `groupArraysAfterLength={1}` to the first example in the dev server.

**Comment by mac-s-g** [comment]
published `react-json-view@1.21.3`

thanks @youngbob for the fix!
